sale_credit_point/models/sale_order.py

```diff
@@ -29,8 +30,10 @@ def credit_point_decrease_msg(self):
     @api.multi
     def action_confirm(self):
         """Check credit before confirmation, update credit if check passed."""
-        for sale in self:
-            sale.credit_point_check()
-            sale.partner_id.credit_point_decrease(
-                sale.amount_total, comment=self.credit_point_decrease_msg)
+        install_module = tools.config.get('init')
```

Comment:
please, put a comment here saying why

Reply:
@leemannd @simahawk I worry about something now.
If you run `odoo -i whatever`, won't tools.config['init'] contains `whatever` until odoo is restarted? Which means the check would be skipped for the entire life of the process. Maybe something empties this key but I couldn't find it.

Reply:
Well tested, it seems it is emptied